### PR TITLE
Add Prettier due to the plugin to order tailwind classes

### PR DESCRIPTION
**Pull request by annanaj**

### CODE REVIEW

File: src/hooks/useFetchNewsData.ts [hunk 1]
```diff
@@ -71,5 +74,5 @@ export default function useFetchNewsData() {
 		data = JSON.parse(cachedData);
 	}
 
-	return { data, loading } as { data: NewsData[], loading: boolean };
+	return { data, loading } as { data: NewsData[]; loading: boolean };
```
Comment (crishpeen) [inline src/hooks/useFetchNewsData.ts:82]: I usually prefer to set return types in the function declaration and have them defined alone. Something like
```
type useFetchNewsDataReturn {
data: NewsData[]; 
loading: boolean
}

export default function useFetchNewsData(): useFetchNewsDataReturn {
```

File: src/pages/NewsList/NewsList.tsx [hunk 2]
```diff
@@ -65,46 +77,61 @@ export default function NewsList() {
...
-										})}
+									<ul className="absolute right-0 top-6">
+										{newsItem.topics
+											.slice(0, 3)
+											.map((topicData) => {
+												let className =
+													'py-1 px-3 mb-1 rounded-l-full shadow-lg text-sm text-white';
+												const topicColor =
+													topicColors[
+														topicData.topic as keyof TopicColors
```
Comment (crishpeen) [inline src/pages/NewsList/NewsList.tsx:151]: image
It would be good to split this into components. Look how it looks in Github :D

**Review by crishpeen — COMMENTED** [review]
oh my, prettier :D

File: src/hooks/useFetchNewsData.ts (hunk 1 above)
Comment (annanaj) [reply src/hooks/useFetchNewsData.ts:82]: as for prettier, do not prefer it, but there is no other plugin to order classes. and also the plugin shows the classes as the sheets are imported (base, components, utils) so margin goes first, display after that.. 👎

**Pull request merged**